Proposals/xgov-37.md

```diff
@@ -0,0 +1,118 @@
+
+This xGov proposal leads to the development of a decentralized NFT lending platform where users can collateralize their NFTs to borrow fungible tokens.
+The platform features two markets: a peer-to-peer market and a peer-to-pool market, offering flexibility and liquidity to participants.
+
+The project’s aim is to design and develop the necessary infrastructure, protocols, and algorithms for the NFT lending platform. Extensive market research, rigorous testing, and optimization efforts will be undertaken to deliver a user-friendly platform with robust security measures.
+
+The development of this NFT lending platform will evolve the ecosystem by unlocking the value of tokenized assets and enabling access to liquidity through borrowing fungible tokens. The tools will be open-sourced, meaning any project can integrate this technology and shape it to their needs. Benefits and revenue will proliferate around Algorand’s network. NFT Lending fosters financial inclusion, empowers NFT holders to leverage their assets, and opens up new opportunities for participants within the tokenization space. The long-term applications of this technology are limited only by how much value, and how many different things, can be tokenized.
+
+## Team
+
+[Benedetto Biondi](https://www.linkedin.com/in/benedetto-biondi-030248199/) is the co-founder and CEO of Folks Finance, one of the leading projects on Algorand, offering a full set of DeFi tools for managing Digital assets. Benedetto has accrued 3 years of experience as a Blockchain Project Manager, is a mentor in the Algorand Miami and Europe Accelerators, a DeFi advisor at Climatecoin, and a DeFi lecturer at the course of Economics and Law for Digital Assets at the University of Florence.
+
+[Gidon Katten](https://www.linkedin.com/in/gidonkatten/) joined Folks Finance as CTO in late 2021 after leaving his job at Amazon. Gidon graduated from Imperial College London where he wrote an award winning paper titled “Issuing Green Bonds on the Algorand Blockchain”.
+
+[Stefano Faieta](https://www.linkedin.com/in/stefanofaieta/) has always been passionate about technology and IT. While pursuing his university degree, he was accepted into exclusive programs such as one focused on startups and innovation, and another on cybersecurity. He graduated with honors in computer science and completed a thesis on the interoperability of smart contracts on the Ethereum network. Initially, he began his career as a frontend developer in traditional finance at banking institutions. However, his persistent desire for experimentation and innovation quickly drove him to merge his two greatest interests - blockchain and frontend technologies.
```

Comment:
External link needs to be formatted using html
eg `<a href="https://www.linkedin.com/in/stefanofaieta/">Stefano Faieta</a>`